components/SectionPage.vue

```diff
@@ -0,0 +1,328 @@
+<template>
+  <div class="section">
+    <header class="c-section__header">
+      <h1 class="c-section__heading u-font--secondary-style u-font--secondary">
+        {{ name }}
+      </h1>
+    </header>
+    <template v-if="!featuredStoriesLoaded">
+      <loading-icon />
+      <v-spacer size="double" />
+    </template>
+    <div class="l-container l-container--14col l-wrap">
+      <section v-if="featuredStories && featuredStories.length === 2">
+        <div class="featured-stories l-grid l-grid--2up l-grid--large-gutters">
+          <v-card
+            v-for="(featuredStory, index) in featuredStories"
+            :key="index"
+            class="gothamist u-space--double--bottom mod-vertical mod-large"
+            :show-gallery-icon="hasGallery(featuredStory.leadAsset)"
+            :image="getArticleImage(featuredStory.leadAsset, featuredStory.ancestry[0].slug, featuredStory.listingImage)"
+            :image-height="150"
+            :image-width="150"
+            :title="featuredStory.title"
+            :title-link="`/${featuredStory.ancestry[0].slug}/${featuredStory.meta.slug}`"
+            :subtitle="featuredStory.description"
+            :tags="featuredStory.sponsoredContent ? [{ name: 'sponsored' }] : []"
+          >
+            <article-metadata
+              :publish-date="!featuredStory.updatedDate ? fuzzyDateTime(featuredStory.meta.firstPublishedAt) : null"
+              :updated-date="featuredStory.updatedDate ? fuzzyDateTime(featuredStory.updatedDate) : null"
+            >
+              <template
+                v-if="featuredStory.legacyId"
+                v-slot:comments
+              >
+                <v-counter
+                  icon="comment"
+                  :value="getCommentCountById(featuredStory.legacyId, moreResultsDisqusData)"
+                  :href="`/${featuredStory.ancestry[0].slug}/${featuredStory.meta.slug}?to=comments`"
+                />
+              </template>
+            </article-metadata>
+          </v-card>
+        </div>
+        <hr
+          class="u-border-accent u-hide-until--m"
+          aria-hidden="true"
+        >
+        <v-spacer size="triple" />
+      </section>
+      <loading-icon v-if="!moreResultsLoaded" />
+      <section v-if="filteredMoreResults">
+        <div
+          v-for="(chunk, chunkIndex) in moreResultsChunks"
+          :key="chunkIndex"
```

Comment:
`chunks` is kind of a loaded term due to legacy cms reasons, could we use something else? would 'pages' work? I guess page can mean a lot of things too but still pagination is pretty common terminology...

Reply:
We both are already using "page" for the page data. How about nugget? LOL

Reply:
I changed it to nugget!